lib/github.atd

```diff
@@ -590,7 +600,7 @@ type repo_issue_event = {
   ?label: base_label option;
   ?assignee: linked_user option;
   ?assigner: linked_user option;
-  ?milestone: milestone option;
+  ?milestone: repo_issue_milestone option;
```

Comment:
```
curl -H "Accept: application/vnd.github.mockingbird-preview" https://api.github.com/repos/moby/moby/issues/34222/timeline
```

indicates that the title-only milestone object is also returned for timeline requests. I'm not sure where you're seeing the `milestone` type used as part of repository events?

Perhaps the type should be called `milestone_reference` or similar? For objects with subsets, I also try to keep that relation fixed through the use of ATD's record inheritance feature. Maybe `milestone` should inherit from `milestone_reference`?